json2html/jsonconv.py

```diff
@@ -78,88 +78,81 @@ def columnHeadersFromListOfDicts(self, inputtedJson):
                 -----------------------------
 
             @contributed by: @muellermichel
-        '''
+        """
 
-        if len(inputtedJson) < 2:
-            return None
-        if not isinstance(inputtedJson[0], dict):
+        if not inputted_json or not isinstance(inputted_json[0], dict):
```

Comment:
Ah so you want lists to always follow the table style, even if only one item is present. Makes sense I think for consistency. The old way just created multiple two-column rows right?